src/main/java/com/amazon/dlic/util/Roles.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.amazon.dlic.util;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Roles {
+    private static final Logger log = LogManager.getLogger(Roles.class);
+
+    public static String[] split(Object rolesObject) {
+        if (rolesObject == null) {
+            return new String[0];
+        } else if (rolesObject instanceof Collection) {
```

Comment:
can you add a sample json here and what will be present in the rolesObject

Reply:
will try to specify the cases with sample JSON